README.md

```diff
@@ -51,214 +123,261 @@ where
 * **MQTT_Library_name** is the MQTT Client library being used.
 * **MQTT_Library_version** is the version of the MQTT Client library being used.
 
-## Branches
+## Versioning
 
-### master branch
-The [master](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/master) branch hosts the continuous development of the AWS IoT Embedded C SDK (C-SDK) libraries. Please be aware that the development at the tip of the master branch is continuously in progress, and may have bugs. Consider using the [tagged releases](https://github.com/aws/aws-iot-device-sdk-embedded-C/releases) of the C-SDK for production ready software.
+C-SDK releases will now follow a date based versioning scheme with the format YYYYMM.NN, where:
 
-### v4_beta_deprecated branch (Formerly named v4_beta)
-The [v4_beta_deprecated](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v4_beta_deprecated) branch contains a beta version of the C-SDK libraries, which is now deprecated. This branch was earlier named as *v4_beta*, and was renamed to *v4_beta_deprecated*. The libraries in this branch will not be released. However, critical bugs will be fixed and tested. No new features will be added to this branch.
+* Y represents the year.
+* M represents the month.
+* N represents the release order within the designated month (00 being the first release).
 
+For example, a second release in June 2021 would be 202106.01. Although the SDK releases have moved to date-based versioning, each library within the SDK will still retain semantic versioning. In semantic versioning, the version number itself (X.Y.Z) indicates whether the release is a major, minor, or point release. You can use the semantic version of a library to assess the scope and impact of a new release on your application.
 
 ## Releases
 
-All of the released versions of the C-SDK libraries are available as git tags. For example, the last release of the v3 SDK version is available at [tag 3.0.1](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.0.1).
+All of the released versions of the C-SDK libraries are available as git tags. For example, the last release of the v3 SDK version is available at [tag 3.1.2](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/v3.1.2).
 
-## Versioning
+### 202011.00
 
-C-SDK releases will now follow a date based versioning scheme with the format YYYYMM.NN, where:
+[API documentation of 202011.00 release](https://docs.aws.amazon.com/embedded-csdk/202011.00/lib-ref/index.html)
 
-* Y represents the year.
-* M represents the month.
-* N represents the release order within the designated month (00 being the first release).
+This release includes new HTTP client, AWS IoT Device Defender, and AWS IoT Jobs libraries. Additionally, there is an update to the coreJSON API. We continue to maintain our code quality criteria.
```

Comment:
This release includes refactored HTTP client, AWS IoT Device Defender, and AWS IoT Jobs libraries. Additionally, there is a major update to the coreJSON API. All libraries continue to undergo code quality checks (e.g. MISRA-C compliance), Coverity static analysis, and validation of memory safety with the C Bounded Model Checker (CBMC) automated reasoning tool.